tests/regressions/builtin_redefinition.c3

```diff
@@ -2,4 +2,6 @@ typedef int : i8
 
 /// @COMPILE; EXPECT ERR
 /// File '*.c3', line 1, in 'typedef'
-/// Error: multiple definitions of type 'int'
+/// Error: type 'int' is ambiguous. Equally good candidates are:
```

Comment:
I think the old error message here is actually better

Reply:
Yeah, me too. I couldn't think of an error message that looked good for both types and functions tho... I've just split it in two now (RIP code reuse).